### ISSUE TITLE
Consider adding a non-magical Darkness Source property to lights, that acts as "normal" non-illuminated space

### ISSUE THREAD

**Issue by unsoluble** [foundryvtt/foundryvtt#10659]
### User Experience

Currently all `Darkness Source` lights have the properties of a _magical_ field of darkness, that fully blocks normal token vision from both outside and within.

While this is a desirable configuration sometimes, it would also be useful to be able to create a `Darkness Source` that is simply an _overriding lack of illumination_ in the radius.

For example, this token with no Vision Range would see a patch of darkness eating into the otherwise-illuminated room:
![CleanShot 2024-04-18 at 16 42 27](https://github.com/foundryvtt/foundryvtt/assets/801582/d0300fbd-3dbe-4525-ae96-40544c7e86a7)

Whereas this token that has a 60' Vision Range should be able to see right through this (if we don't want it to be a magical blockage):
![CleanShot 2024-04-18 at 16 43 36](https://github.com/foundryvtt/foundryvtt/assets/801582/62dc569b-4dac-42e9-bfb1-c8218a93c271)

**Comment by aaclayton** [comment]
This is a tricky one because standard non-magical darkness does not have a point source. We are adding in v12 scene region behaviors which support this. In Dev2 we added a Darkness Modification region behavior which can alter the darkness level in a particular area of the scene. In our next testing phase there will be a Global Illumination suppression region behavior which can exclude global illumination from a certain area of the scene. 

These combination of region behaviors are the intended path forward towards accommodating the type of gameplay you have in mind here, although it's possible I misunderstand some aspect of what you want.

**Comment by Feu-Secret** [comment]
I think that the `Adjust Darkness Level` behavior for scene region is exactly what you need. In particular, the "Darken" mode to create less illuminated areas (kind of shadow patches). Example with the house shadow projection, reducing the darkness level in this specific area.
![image](https://github.com/foundryvtt/foundryvtt/assets/66535080/36e35eae-1e5f-48a1-bdff-1d54aeba55c2)

**Comment by unsoluble** [comment]
I think part of the confusion here rooted in the extreme overloading of the term "Darkness" on the platform. We now have:

1. Darkness, the aesthetic scene overlay (or Region-defined overlay), which makes things _look_ darker but doesn't affect illumination (in terms of token vision).
2. Darkness, the light source type, which is a hard-walled lack of illumination that doesn't interact with token vision at all, except to block it.
3. Darkness, the concept of "is not currently illuminated".

What I was musing about in the ticket here was 3. :)

**Comment by Norc** [comment]
I think adding the upcoming #10390 behavior to the Scene Region would add the Darkness (3) to the area as well. Darkness (1) can already be added by the Adjust Darkness Level behavior and is a good solution. You can add both behaviors to the same scene region, which should do it, I think.

I'll think about what, if anything, can be done to improve communication of the 3 different types of darkness that you accurately describe.

**Comment by unsoluble** [comment]
Yeah I think the (more important?) meta issue here is disambiguating the Darkness term. I think if the platform is going to commit to retaining it for "aesthetic dimness", then it would be best to figure out some other word to describe what these point-source lights are doing.
